diff --git a/node_modules/@catlair/bilicomic-dataflow/bilicomic_dataflow.js b/node_modules/@catlair/bilicomic-dataflow/bilicomic_dataflow.js
index e48bf3f..aec8e7a 100644
--- a/node_modules/@catlair/bilicomic-dataflow/bilicomic_dataflow.js
+++ b/node_modules/@catlair/bilicomic-dataflow/bilicomic_dataflow.js
@@ -1,7 +1,7 @@
 let imports = {};
 imports['__wbindgen_placeholder__'] = module.exports;
 let wasm;
-const { fetch } = require(String.raw`./snippets/bilicomic-dataflow-eb673a8c566a4244/defined-in-js.js`);
+const { fetch } = require(`./snippets/bilicomic-dataflow-eb673a8c566a4244/defined-in-js.js`);
 const { TextDecoder, TextEncoder } = require(`util`);
 
 const heap = new Array(32).fill(undefined);
